docker/Dockerfile

```diff
@@ -58,19 +55,21 @@ ENV RUST_BACKTRACE=1 \
 ARG RUST_VERSION=stable
 RUN RUST_VERSION=$RUST_VERSION curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain $RUST_VERSION -y
 
-RUN rustup component add --toolchain $RUST_VERSION rustfmt && cargo install cargo-release
+RUN rustup component add --toolchain $RUST_VERSION rustfmt \
+  && cargo install cargo-release \
+  && cargo install --force cbindgen
 
 FROM system-setup AS build-suricata
 
-WORKDIR $BUILD_DIR
+WORKDIR /build
 
 #Checkout suricata
-ARG SURICATA_VERSION=5.0.0
+ARG SURICATA_VERSION=6.0.0
 RUN SURICATA_VERSION=$SURICATA_VERSION git clone https://github.com/dbcfd/suricata
```

Comment:
Linking DBs Suricata MR, that will allow us to use mainline suricata here:
https://github.com/OISF/suricata/pull/5489